src/app/app.metareducers.ts

```diff
@@ -33,6 +32,6 @@ export function universalMetaReducer(reducer) {
 }
 
 export const appMetaReducers = [
-  // debugMetaReducer,
+  debugMetaReducer,
```

Comment:
Any particular reason for re-enableing this? 
I made that metareducer to have some idea what happened to the store when @ngrx/store-devtools had an issue with the angular 4 router. But that issue is fixed now. I left the debug metareducer there, commented out, in case we should need it again.

Reply:
No reason. Maybe the debug metareducer should be conditionally added from the debug configuration property.

Reply:
That's fine